framework/src/main/java/org/checkerframework/framework/source/SourceChecker.java

```diff
@@ -356,7 +356,10 @@
 
     // Whether to output resource statistics at JVM shutdown
     // org.checkerframework.framework.source.SourceChecker.shutdownHook()
-    "resourceStats"
+    "resourceStats",
+
+    // Parse all JDK files at startup rather than as needed.
+    "parseAllJdk"
```

Comment:
This new option should be documented in the manual.

Reply:
This was done in #3279.